src/libvirt-xml-create.js

```diff
@@ -263,15 +263,23 @@ export function getSnapshotXML(name, description, disks, memoryPath, isExternal,
 
         const disksElem = doc.createElement('disks');
         disks.forEach(disk => {
-            // Disk can have attribute "snapshot" set to "no", which means no snapshot should be created of the said disk
-            // This cannot be configured through cockpit, but we should uphold it nevertheless
-            // see "snapshot" attribute of <disk> element at https://libvirt.org/formatdomain.html#hard-drives-floppy-disks-cdroms
-            if (disk.snapshot !== "no") {
-                const diskElem = doc.createElement('disk');
-                diskElem.setAttribute('name', disk.target);
-                diskElem.setAttribute('snapshot', 'external');
-                disksElem.appendChild(diskElem);
-            }
+            // Disk can have attribute "snapshot" set to "no", which
+            // means no snapshot should be created of the said disk
+            // This cannot be configured through cockpit, but we
+            // should uphold it nevertheless see "snapshot" attribute
```

Comment:
You dropped the line break between "nevertheless" and "see", this is now a bit hard to read. Add a `. ` or move it back to its own line?

Reply:
Fixed.